app/models/transaction.rb

```diff
@@ -29,6 +31,21 @@ def actualized?
 
   private
 
+  def source_account_validation
+    is_income = transaction_type_id == TransactionType.income.id
+    is_salary = transaction_type_id == TransactionType.salary.id
+
+    return unless is_income || is_salary
```

Comment:
Change to 
```
return if transaction_type_id == TransactionType.initialize.id
```

It seems to me that initialization is the only valid transaction that has no source.  Everything else requires a source.